src/connectivity.jl

```diff
@@ -285,13 +285,14 @@ from `v`.
 - `dir=:out`: If `g` is directed, this argument specifies the edge direction
 with respect to `v` of the edges to be considered. Possible values: `:in` or `:out`.
 """
-function neighborhood(g::AbstractGraph, v::Integer, d::Integer; dir=:out)
+neighborhood(g::AbstractGraph, v::Integer, d::Integer; dir=:out) =
```

Comment:
This is redundant with BFS code should this be a `bfs(TruncatedBFS(d), g, v; dir=:out)`